appveyor.yml

```diff
@@ -3,17 +3,20 @@ environment:
     - nodejs_version: '8'
     - nodejs_version: '6'
     - nodejs_version: '4'
+
 install:
   - ps: Install-Product node $env:nodejs_version
   - set CI=true
   - npm install --global npm@latest
   - set PATH=%APPDATA%\npm;%PATH%
   - npm install
-matrix:
-  fast_finish: true
-build: off
-shallow_clone: true
```

Comment:
why you remove these lines?

Reply:
Shallow clone was causing issues. See https://github.com/okonet/lint-staged/pull/255#issuecomment-327121582

Reply:
I must have removed the `fast_finish` by mistake. But would prefer keeping it off. Would be useful to know if it is an issue specific to any one node version.